teams/hubturbo.md

```diff
@@ -30,7 +30,7 @@ permalink: /teams/hubturbo/
 
  - [@xinan](https://github.com/xinan)
    ([Liu Xinan](http://xinan.io))
-   - Technology Expertise:
+   - Technology Expertise: Performance Optimization
```

Comment:
Whichever topic you choose, plan long-term. The goal is to say 'I'm an expert of X' when you apply for jobs in 2-3 years time.
1. Choose something that you don't mind working on for at least few more years.
2. Choose something that you think the potential employers will value.